<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMonthPicker</class>
 <widget class="QWidget" name="WidgetMonthPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/compass.png</normaloff>:/icons/icons/compass.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Start Date</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxStartMonth"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxStartYear"/>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxEndMonth"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxEndYear"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>End Date</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
